<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>621</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_Place">
       <attribute name="icon">
        <iconset resource="../../Server1.0/Project_Server/Icons.qrc">
         <normaloff>:/Icons/Place.png</normaloff>:/Icons/Place.png</iconset>
       </attribute>
       <attribute name="title">
        <string>地点</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_Canteen">
          <property name="text">
           <string>食堂</string>
          </property>
          <property name="icon">
           <iconset resource="../../Server1.0/Project_Server/Icons.qrc">
            <normaloff>:/Icons/Canteen.png</normaloff>:/Icons/Canteen.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_Dorm">
          <property name="text">
           <string>宿舍</string>
          </property>
          <property name="icon">
           <iconset resource="../../Server1.0/Project_Server/Icons.qrc">
            <normaloff>:/Icons/Dorm.png</normaloff>:/Icons/Dorm.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_Library">
          <property name="text">
           <string>图书馆</string>
          </property>
          <property name="icon">
           <iconset resource="../../Server1.0/Project_Server/Icons.qrc">
            <normaloff>:/Icons/Library.png</normaloff>:/Icons/Library.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_Basketball">
          <property name="text">
           <string>篮球场</string>
          </property>
          <property name="icon">
           <iconset resource="../../Server1.0/Project_Server/Icons.qrc">
            <normaloff>:/Icons/Basketball.png</normaloff>:/Icons/Basketball.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_MyInfo">
       <attribute name="icon">
        <iconset resource="../../Server1.0/Project_Server/Icons.qrc">
         <normaloff>:/Icons/MyInfo.png</normaloff>:/Icons/MyInfo.png</iconset>
       </attribute>
       <attribute name="title">
        <string>个人</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>性别:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>姓名:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_Gender">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_Name">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_Photo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>个性签名</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPlainTextEdit" name="plainTextEdit_Signature">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMyInfo">
   <property name="text">
    <string>个人</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
